### PR TITLE
feat: Support revoked PVCs and add custom debugging logger

### PR DESCRIPTION
**Pull request by AfsarFresh**
SMART Health Card has implemented a new feature called a [revocation ID](https://secure-web.cisco.com/1wMLn7TDareBON-xhR1V06zWhhzFDoyuyGT-x4CQEC-0fOohkmKyDnWlP8ouR81OPGJ8tImr8-cmmJ9J3haXqC3AQnnhgplg1sad-tV0fPEQD9xtbH1UW-sLNoXYqZdV3uy8RRpCVMtsSaZF_gEBPs3ox5PlQoP5bgOJW32bk5Kh-fc40-0tDP3eFxb1aAqUPyfW-WMHt6kJBSUIYKvVQbItcPyIgz1qoIeM4sjQNRlCv8_F7LEgJdNa8Hf7yBKFHzYMFdZzQtcH9gP8jTvWzngTfDvH4MvwOKDwQxhK-_iX6NxhkrCfButdalubCIRIe/https%3A%2F%2Fspec.smarthealth.cards%2F%23revocation) (a unique identifier for each PVC). The unique identifier will allow provinces and territories to de-authorize (or revoke) select PVCs (e.g., for fraudulent reasons). The rid values may also be timeboxed. Similar to the approved issuers, each province/territory will have a list of unique identifiers (new endpoint) that will read as ‘Does not meet requirements'.

### CODE REVIEW

File: BCVaccineValidator/Classes/Services/RevocationManager.swift [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    func downloadAndCacheIfNeeded(completion: @escaping () -> Void) {
+        // Get list of issuers
+        IssuerManager.shared.getIssuers { [weak self] result in
+            guard let self = self else { return }
+            guard let issuers = result else {
+                // It is highly unlikely that result is nil.
+                Logger.logFailure("Critical Error: No issuers found") // NO I18N
+                return completion()
+            }
+            let displatchGroup = DispatchGroup()
```
Comment (freshfawzy) [inline BCVaccineValidator/Classes/Services/RevocationManager.swift]: there is a typo here, please rename "displatchGroup" to "dispatchGroup"

**Review by freshfawzy — APPROVED** [review]
awesome work!
just one typo :) 
Approving

File: BCVaccineValidator/Classes/Services/RevocationManager.swift (hunk 1 above)
Comment (AfsarFresh) [reply BCVaccineValidator/Classes/Services/RevocationManager.swift]: Will fix

**Comment by AfsarFresh** [comment]
> awesome work! just one typo :) Approving
Thanks, Fawzy.

**Pull request merged**